core/testcontainers/core/config.py

```diff
@@ -125,9 +128,19 @@ def docker_auth_config(self, value: str) -> None:
         self._docker_auth_config = value
 
     def tc_properties_get_tc_host(self) -> Union[str, None]:
+        if "tc_properties_get_tc_host" in _WARNINGS:
+            warning(_WARNINGS.pop("tc_properties_get_tc_host"))
         return self.tc_properties.get("tc.host")
 
     @property
+    def tc_properties_tc_host(self) -> Union[str, None]:
+        return self.tc_properties.get("tc.host")
+
+    @property
+    def tc_properties_testcontainers_reuse_enable(self) -> bool:
+        enabled = self.tc_properties.get("testcontainers.reuse.enable")
+        return enabled == "true"
+
     def timeout(self) -> int:
```

Comment:
I just noticed that `timeout` here had the `@property` decorator before. Was it removed by accident?

Reply:
Yes, this was an accident.